torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -957,6 +957,8 @@ std::pair<NvrtcFunction, std::string> nvrtcCompile(
   }
 #else
   if (disable_fma) {
+    TORCH_WARN(
```

Comment:
Would it make more sense to have this warning in `isDisabled()`?

Reply:
Good idea, that's also future proof in case we are using it somewhere else. :+1: